executor/runtime/types/types.go

```diff
@@ -63,7 +61,8 @@ type GPUContainer interface {
 
 // Container contains config state for a container.
 // It is not safe to be used concurrently, synchronization and locking needs to be handled externally.
-type Container struct { // nolint: maligned
+type Container struct {
+	// nolint: maligned
```

Comment:
`gofmt` has been doing this for me too (breaking the `nolint` to a different line). Does this break the linter?

Reply:
It just means you have to write no lint in every "paragraph" -- It might be new behaviour on go 1.10?